src/main/java/io/netty/incubator/codec/http3/Http3.java

```diff
@@ -52,6 +59,77 @@ static void setLocalControlStream(Channel channel, QuicStreamChannel controlStre
         channel.attr(HTTP3_CONTROL_STREAM_KEY).set(controlStreamChannel);
     }
 
+    private static final AttributeKey<Supplier<? extends ChannelHandler>> HTTP3_CODEC_SUPPLIER =
+            AttributeKey.valueOf(Http3.class, "HTTP3CodecSupplier");
+
+    /**
+     * Returns the {@link Supplier} that should be used to obtain a HTTP/3 codec for the underlying connection.
+     *
+     * Usually you as an end-user you just want to use {@link Http3RequestStreamInitializer} directly.
```

Comment:
```suggestion
     * Usually you as an end-user you just want to use {@link Http3RequestStreamInitializer} directly.
```
As an end-user in most cases you just want to use {@link Http3RequestStreamInitializer} directly.